sourcemash/frontend/assets.py

```diff
@@ -39,8 +39,10 @@
                 "js/views/feed_view.js",
                 "js/views/saved_view.js",
                 "js/views/browse_view.js",
+                "js/views/register_view.js",
```

Comment:
What happens if I'm already a user on the site and accidentally upvote before logging in? The modal pops up to register. Do I need to close the modal first in order to sign in? Maybe we should at least have an "I already have an account link" on the modal which can route to a more explicit login view, like the one you made for register?